SpatialGDK/Source/SpatialGDK/Public/SpatialView/DispatcherInterface.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/Callbacks.h"
+#include "SpatialView/ViewDelta.h"
+
+namespace SpatialGDK
+{
+struct FEntityComponentChange
+{
+	Worker_EntityId EntityId;
+	const ComponentChange& Change;
+};
+
+using FEntityCallback = TCallbacks<Worker_EntityId>::CallbackType;
+using FComponentValueCallback = TCallbacks<FEntityComponentChange>::CallbackType;
+
+class IDispatcher
```

Comment:
Nit - I think elsewhere we've used AbstractX (although my preference is always Dispatcher for the base and something more specific for the derived).
But I don't really care that much. Up to you if this is actually important enough to change.

Reply:
Actually if we are doing FStuff etc then I could be convinced we should change other things to IStuff too.